packages/react-components/src/templates/TagsPageBody.tsx

```diff
@@ -1,5 +1,13 @@
-import { CRNTagSearchEntities } from '@asap-hub/algolia';
-import { TagSearchResponse } from '@asap-hub/model';
+/* eslint-disable no-underscore-dangle */
```

Comment:
I don't think this `/* eslint-disable no-underscore-dangle */` is necessary, is it?

Reply:
https://github.com/yldio/asap-hub/actions/runs/7018626145/job/19094459725?pr=3959